packages/slice-machine/src/features/slices/sliceCards/SharedSliceCard.tsx

```diff
@@ -0,0 +1,227 @@
+import {
+  Box,
+  Button,
+  Checkbox,
+  DropdownMenu,
+  DropdownMenuContent,
+  DropdownMenuItem,
+  DropdownMenuTrigger,
+  Icon,
+  IconButton,
+  Text,
+  tokens,
+} from "@prismicio/editor-ui";
+import Link from "next/link";
+import type { FC } from "react";
+
+import { ComponentUI } from "@lib/models/common/ComponentUI";
+import { Card, CardFooter, CardMedia, CardStatus } from "@src/components/Card";
+import { countMissingScreenshots, getScreenshotUrl } from "@src/domain/slice";
+import {
+  StatusBadge,
+  type StatusBadgeProps,
+} from "@src/features/changes/StatusBadge";
+import { SLICES_CONFIG } from "@src/features/slices/slicesConfig";
+import { AddPhotoAlternateIcon } from "@src/icons/AddPhotoAlternateIcon";
+
+type SharedSliceCardProps = {
```

Comment:
⚠️ I REALLY like the change, easier to use BUT, and I like my but, because I know it's something you want to improve.

I feel like it's too hard to understand what type of props you need to set in order to have the desired result.
Could it be possible to have a comment explaining the typically / possible use cases?

I'm mostly talking for variant / mode I guess so